linux/rpi-imager.metainfo.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!--Copyright 2020 Flathub maintainers-->
+<component type="desktop-application">
+  <id>org.raspberrypi.rpi-imager</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>Apache-2.0</project_license>
+  <name>Raspberry Pi Imager</name>
+  <summary>Raspberry Pi imaging utility</summary>
+  <description>
+    <p>
+      Raspberry Pi Imager downloads a .JSON file from the Raspberry Pi
+      website with a list of all current download options, ensuring you are
+      always installing the most up-to-date version.
+    </p>
+    <p>
+      Once you’ve selected an operating system from the available options,
+      the utility reads the relevant file directly from the Raspberry Pi
+      website and writes it straight to the SD card. This speeds up the
+      process quite considerably compared to the standard process of reading
+      it from the website, writing it to a file on your hard drive, and then,
+      as a separate step, reading it back from the hard drive and writing it
+      to the SD card.
+    </p>
+    <p>
+      During this process, Raspberry Pi Imager also caches the downloaded
+      operating system image – that is to say, it saves a local copy on your
+      computer, so you can program additional SD cards without having to
+      download the file again.
+    </p>
+  </description>
+  <launchable type="desktop-id">rpi-imager.desktop</launchable>
+  <screenshots>
+    <screenshot type="default">
+      <image>http://downloads.raspberrypi.org/imager/IMAGING-UTILITY-MAIN.png</image>
+      <caption>Main window</caption>
+    </screenshot>
+    <screenshot>
+      <image>http://downloads.raspberrypi.org/imager/IMAGING-UTILITY-OS.png</image>
+      <caption>Choose OS</caption>
+    </screenshot>
+    <screenshot>
+      <image>http://downloads.raspberrypi.org/imager/IMAGING-UTILITY-SD.png</image>
+      <caption>Choose SD</caption>
+    </screenshot>
+    <screenshot>
+      <image>http://downloads.raspberrypi.org/imager/IMAGING-UTILITY-WRITE.png</image>
+      <caption>Write in progress</caption>
+    </screenshot>
+    <screenshot>
+      <image>http://downloads.raspberrypi.org/imager/IMAGING-UTILITY-DONE.png</image>
+      <caption>Write done</caption>
+    </screenshot>
+  </screenshots>
+  <url type="homepage">https://github.com/raspberrypi/rpi-imager</url>
+  <provides>
+    <binary>rpi-imager</binary>
+  </provides>
+  <releases>
+    <release version="1.2" date="2020-03-10" />
```

Comment:
I'm guessing this would have to be updated on every release?

Reply:
Some upstream provide detailed release notes through the release metadata (so that changes are displayed and formatted nicely in software stores), others just keep one entry and only update the version and date (which is comparatively simpler to maintain).

Your call.